src/children.js

```diff
@@ -8,7 +8,7 @@ export default function children(node) {
 		return node.flatMap(node => children(node));
 	}
 
-	return childProperties.filter(property => node[property]).map(property => node[property]);
+	return childProperties.filter(property => node[property]).flatMap(property => node[property]);
```

Comment:
Are you sure this will work? I `flatMap()` flattens after mapping, whereas it seems like this would need the opposite?
I.e. this:

```suggestion
	return childProperties.filter(property => node[property]).flat().map(property => node[property]);
```

Reply:
It should flatten after mapping I'm pretty sure. I actually already tested both out, and for a given input, here is what `.flat().map` produces:

```js
[
  [
    { type: 'CallExpression', arguments: [Array], callee: [Object] },
    { type: 'Literal', value: 'external', raw: "'external'" }
  ],
  { type: 'Identifier', name: 'if' }
]
```

And here is what `flatMap` produces:

```js
[
  {
    type: 'CallExpression',
    arguments: [ [Object], [Object] ],
    callee: { type: 'Identifier', name: 'starts' }
  },
  { type: 'Literal', value: 'external', raw: "'external'" },
  { type: 'Identifier', name: 'if' }
]
```